### ISSUE TITLE
Duplicate meter in DHARMA_prosodicPatterns_v01.xml

### ISSUE THREAD

**Issue by michaelnmmeyer** [erc-dharma/project-documentation#228]
@danbalogh In DHARMA_prosodicPatterns_v01.xml, we have two meters named paṅkti which do not refer to the same phonetic pattern. If this is normal, we need to distinguish them. I propose to add an index number at the end of the word, as in paṅkti1 and paṅkti2, to keep things simple.

**Comment by danbalogh** [comment]
Hmm, not an easy one. Thanks for highlighting the problem. I am very strongly against paṅkti1 and paṅkti2 if it can be avoided at all. It turns out that it can, now that I've thought about it, but let me keep my thought process here.
In fact, paṅkti can apparently be no fewer than 3 different things in this context:

1. The name of a specific prosodic template of 5 syllables;
2. The name of a class of prosodic templates with 10 syllables;
3. The name of a specific prosodic template of 10 syllables.

The fact that 2 exists is not much of a problem for us, because class names are encoded as `<label type="generic">` and so encoding paṅkti as a class name would not cause conflict in the lookup procedure.
The fact that 3 exists is a problem, as pointed out in my comment under the 10-syllable paṅkti item (comment probably from december 2021, as per the changelog), though at that point I was not aware of the existence of 1, and we did not yet have a finalised solution for 2, so the comment is not completely relevant now.
But having thought about it, it seems to me that 3 is a phantom. Given the way `<label type="generic">` is now used in the file, the 10-syllable paṅkti entry in the prosodic patterns xml file is in my opinion not correctly encoded, and it should instead be:

```
        <item>
          <measure unit="syllable">10</measure>
          <label type="generic" xml:lang="san-Latn">paṅkti</label>
          <name/>
          <seg type="xml">--------+=</seg>
          <seg type="prosody">⏑⏑⏑⏑⏑⏑⏑⏑–⏓</seg>
	<seg type="gana">na-na-sa-ga</seg>
	<listBibl>
		<bibl><ptr target="bib:Apte1890_01"/><citedRange unit="appendix">A: Sanskrit Prosody</citedRange><!--not in Apte--></bibl>
		<bibl><ptr target="bib:Velankar1949_01"/></bibl>
		<bibl><ptr target="bib:Zoetmulder1974_01"/></bibl>
	</listBibl>
</item>

```

The above should replace lines 2911 to 2928 of the xml file as of now.

If this is accepted, then the corpus must be checked for occurrences of `@met="paṅkti"`, and if any of those have 10 syllables per line, then they should be corrected to `@met="--------+="`. There will be very few occurrences (or none at all) of any kind of paṅkti in the corpus, so this is not an unreasonable task.

Just in case, it may be best to involve @arlogriffiths and have his OK before accepting my solution.
@arlogriffiths : it took me some time to work out what was going on here, so I summarise the issue in hopes that I can save you that time. Someone, long ago, added an item for a 10-syllable metre called paṅkti to the prosodic template. I don't know where this metre occurred, but I assume it was in an actual text, since this particular pattern is not listed in Zoetmulder's appendix. I further assume that the name "paṅkti" for this metre was not found in a metrical treatise for that particular template, but rather whoever encoded the entry used the class name as the name of the metre. I am quite sure that all this happened before we finalised the system of using `<label type="generic">` to record class names for metres that we cannot put a name to (email exchanges in March 2022). So we can now in retrospect encode the entry correctly, with the generic label paṅkti and the raw prosodic pattern for a name.
